fractions/javaee/jaxrs-jsonb/README.adoc

```diff
@@ -0,0 +1,4 @@
+= JAX-RS + JSON-B
+
+Provides support within JAX-RS application for JSON Binding processing
```

Comment:
Make it a single line (1 sentence = 1 line) and probably remove the word "processing".